rusoto/services/acm-pca/src/generated.rs

```diff
@@ -153,6 +153,7 @@ pub struct CertificateAuthorityConfiguration {
 }
 
 #[derive(Default, Debug, Clone, PartialEq, Serialize)]
+#[cfg_attr(feature = "deserialize_structs", derive(Deserialize))]
```

Comment:
TIL derives can be set on multiple lines like this. [Nifty!](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=7b811cfa5ef353b879dd9378f37cca72) 😄